lib/media/track/remotevideotrack.js

```diff
@@ -105,7 +105,10 @@ class RemoteVideoTrack extends RemoteMediaVideoTrack {
    */
   _start(dummyEl) {
     const result = super._start.call(this, dummyEl);
-    updateRenderHints(this);
+    // after emitting started, turn off track if not needed.
+    if (this._subscribedTrackSwitchOffMode === 'auto') {
```

Comment:
Is this check necessary since you are performing the same check within `updateRenderHints()`. Another nitpick: if there is a chance that render hints are not updated when you call `updateRenderHints()`, it should be called `maybeUpdateRenderHints()`.

Reply:
this check is necessary because `updateRenderHints` is no-op only when both `subscribedTrackSwitchOffMode != auto` and  `contentPreferencesMode != auto` 
Good Idea about renaming `maybeUpdateRenderHints` - I will make the change